cardstack/src/screens/PayMerchant/usePayMerchant.ts

```diff
@@ -217,12 +217,37 @@ export const usePayMerchant = () => {
     currencyConversionRates,
   ] = useNativeCurrencyAndConversionRates();
 
+  const spendAmount = useMemo(
+    () =>
+      convertToSpend(
+        convertStringToNumber(inputValue || '0'),
+        nativeCurrency,
+        currencyConversionRates[nativeCurrency]
+      ),
+    [currencyConversionRates, inputValue, nativeCurrency]
+  );
+
+  const sortedPrepaidCards: PrepaidCardType[] = useMemo(
+    () =>
+      []
+        .concat(prepaidCards || [])
```

Comment:
```suggestion
      [...prepaidCards]
```

Would this be a more succinct alternative?